.travis.yml

```diff
@@ -5,12 +5,21 @@ python:
   - 3.6
   - 3.5
   - 3.4
+services:
+  - mysql
+  - postgresql
 
 # installing dependencies
 before_install:
   - nvm install 6; nvm use 6
   - npm install
   - npm install -g configurable-http-proxy
+  - |
+    if [[ $JUPYTERHUB_TEST_DB_URL == mysql* ]]; then
```

Comment:
> I would highly recommend using pymysql rather than mysqlclient

I tried that first, but ran into latin-1 encoding issues in the unicode tests. It does seem preferable in general, though.